### PR TITLE
sha-256 as MTI

### PR DESCRIPTION
**Pull request by Sakurann**
Issue #50

### CODE REVIEW

File: main.md [hunk 1]
```diff
@@ -301,6 +301,8 @@ Algorithm" registry [IANA.Hash.Algorithms]. SD-JWTs with hash algorithm
 identifiers not found in this registry are not considered valid and MUST NOT be
 accepted by verifiers.
 
+To promote interoperability among implementations, the SHA-256 hash algorithm is mandatory to implement.
```
Comment (peppelinux) [inline main.md:304]: ... the SHA-256 hash algorithm MUST be implemented.

**Review by peppelinux — CHANGES_REQUESTED** [review]

File: main.md (hunk 1 above)
Comment (PieterKas) [reply main.md:304]: Agreed. Perhaps something even shorter like:

Implementations MUST support the SHA-256 hash algorithm as shown in Example 1.

Comment (PieterKas) [reply main.md:304]: Agreed with @peppelinux proposal. It may be sufficient to just say:

Implementations MUST support the SHA-256 hash algorithm ()

Comment (Sakurann) [inline main.md:304]:
```suggestion
To promote interoperability, implementations MUST support the SHA-256 hash algorithm.
```

**Review by peppelinux — APPROVED** [review]

**Pull request merged**